src/util/execCommand.js

```diff
@@ -13,6 +13,10 @@ const isWindows = os.platform() === 'win32';
 function execCommand(exec, command, ...args) {
 	exec = exec || spawnSync;
 	command = isWindows ? `${command}.cmd` : command;
+	args[0] = isWindows
```

Comment:
Let's extract this to its own function with a meaningful name, comments explaining why we need this and unit tests.